### PR TITLE
GnuPG Dependencies

### PR DESCRIPTION
**Pull request by yuui-tanabe**
I added pinentry to gnupg 2 dependencies

### CODE REVIEW

**Comment by filnet** [comment]
11 commits for a one line change ?

File: mingw-w64-gnupg/PKGBUILD [hunk 1]
```diff
@@ -74,7 +76,7 @@ build() {
       --enable-gpg2-is-gpg
   
   MSYS2_ARG_CONF_EXCL="-DGNUPG_BINDIR=;-DGNUPG_LIBEXECDIR=;-DGNUPG_LIBDIR=;-DGNUPG_DATADIR=;-DGNUPG_SYSCONFDIR=;-DGNUPG_LOCALSTATEDIR=;-DLOCALEDIR=" \
-  make
+  make -j $(nproc)
```
Comment (lazka) [inline mingw-w64-gnupg/PKGBUILD]: makepkg sets MAKEFLAGS, passing -j is not needed

File: mingw-w64-gnupg/PKGBUILD [hunk 2]
```diff
@@ -86,7 +88,7 @@ check() {
 package() {
   cd "${srcdir}/build-${MINGW_CHOST}"
   MSYS2_ARG_CONF_EXCL="-DGNUPG_BINDIR=;-DGNUPG_LIBEXECDIR=;-DGNUPG_LIBDIR=;-DGNUPG_DATADIR=;-DGNUPG_SYSCONFDIR=;-DGNUPG_LOCALSTATEDIR=;-DLOCALEDIR=" \
-  make -j1 DESTDIR=${pkgdir} install
+  make -j $(nproc) DESTDIR=${pkgdir} install
```
Comment (lazka) [inline mingw-w64-gnupg/PKGBUILD]: same

File: mingw-w64-gnupg/PKGBUILD [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # Maintainer: Alexey Pavlov <alexpux@gmail.com>
+# Maintainer: Yuui Tanabe <yuuitanabe@163.com>
 
 _realname=gnupg
 pkgbase=mingw-w64-${_realname}
```
Comment (lazka) [inline mingw-w64-gnupg/PKGBUILD]: Increase pkgrel

**Pull request closed without merging**